app/src/main/res/values/strings.xml

```diff
@@ -499,6 +499,7 @@
     <string name="no_segments_found">There are no segments for this video yet.</string>
     <string name="start_sleep_timer">Start sleep timer</string>
     <string name="disable_sleep_timer">Disable sleep timer</string>
+    <string name="minutes_left">%d min left</string>
```

Comment:
I think this needs to support [pluralization](https://developer.android.com/guide/topics/resources/string-resource#Plurals), as not every language uses singular abbreviations.